code/src/containers/dashboard-orders-filter-drawer/hooks/use-filtered-admin-orders/useFilteredAdminOrders.test.ts

```diff
@@ -54,6 +54,7 @@ describe("useFilteredAdminOrders", () => {
       new URLSearchParams(),
       mockSetSearchParams
     ]);
+    jest.clearAllMocks();
```

Comment:
Why we need it? We mock everything and then clear all mocks. Maybe we can move it to `afterEach` or remove completely?

Reply:
Moved into AfterEach